help/destinations/catalog/crm/salesforce.md

```diff
@@ -56,35 +56,58 @@ Also ensure that the [scopes](https://help.salesforce.com/s/articleView?id=conne
 
 >[!IMPORTANT]
 >
->Ensure there are no whitespace characters in the field name. Instead, use the underscore `(_)` character as a separator.
+>Do not include whitespace characters in the field name. Instead, use the underscore `(_)` character as a separator.
+>Within [!DNL Salesforce] you must create custom fields with a **[!UICONTROL Field Name]** that exactly matches the value specified within **[!UICONTROL Mapping ID]** for each activated Platform segment. For example the screenshot below shows a custom field named `crm_2_seg`. When activating a segment to this destination, add `crm_2_seg` as **[!UICONTROL Mapping ID]** to populate segment audiences from Experience Platform into this custom field.
+
+An example of custom field creation in [!DNL Salesforce], *Step 1 - Select the data type*, is shown below:
+![Salesforce UI screenshot showing custom field creation, Step 1 - Select the data type.](../../assets/catalog/crm/salesforce/create-salesforce-custom-field-step-1.png)
+
+An example of custom field creation in [!DNL Salesforce], *Step 2 - Enter the details for the custom field*, is shown below:
+![Salesforce UI screenshot showing custom field creation, Step 2 - Enter the details for the custom field.](../../assets/catalog/crm/salesforce/create-salesforce-custom-field-step-2.png)
+
+>[!TIP]
+>
+>* To distinguish between custom fields used for Platform segments and other custom fields within [!DNL Salesforce] you could include a recognizable prefix or suffix when creating the custom field. For example, instead of `test_segment`, use `Adobe_test_segment` or `test_segment_Adobe`
+>* If you already have other custom fields created in [!DNL Salesforce], you can use the same name as the Platform segment, to easily identify the segment in [!DNL Salesforce].
 
 >[!NOTE]
 >
 >* Objects in Salesforce are restricted to 25 External fields, see [Custom Field Attributes](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5).
 >* This restriction implies that you can only have a maximum of 25 Experience Platform segment memberships active at any time. 
->* If you have reached this limit within Salesforce, you must to remove the custom attribute from Salesforce that were used to store the segment status against older segments within Experience Platform before a new **[!UICONTROL Mapping ID]** can be used.
+>* If you have reached this limit within Salesforce, you must remove the custom attribute from Salesforce that were used to store the segment status against older segments within Experience Platform before a new **[!UICONTROL Mapping ID]** can be used.
```

Comment:
Minor, I think we should use plural: 

[...] you must remove the custom attributes from Salesforce that were used [...]